s3transfer/download.py

```diff
@@ -131,6 +131,27 @@ def _get_temp_fileobj(self):
         return f
 
 
+class DownloadSeekableOutputManager(DownloadOutputManager):
+    @classmethod
+    def is_compatible(self, download_target):
+        return (
+            hasattr(download_target, 'seek') and
+            hasattr(download_target, 'tell')
```

Comment:
seekable() is not available on the fileobj returned from ``open()`` in Python 2. How about I just check that tell() does not return an IOError?